jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -545,8 +545,25 @@ def _plot_uncertainties(self):
                 self.figure.marks = list(self.figure.marks) + [error_line_mark]
 
     def set_plot_axes(self):
-        # Set axes labels for the spectrum viewer
+        # Set y axes labels for the spectrum viewer
+        y_display_unit = self.state.y_display_unit
+        y_unit = u.Unit(y_display_unit) if y_display_unit else u.dimensionless_unscaled
+
+        # default to Flux Density for flux density or uncaught types
         flux_unit_type = "Flux density"
+
+        if y_unit.is_equivalent(u.Jy):
+            flux_unit_type = "Flux density"
+        elif y_unit.is_equivalent(u.Jy / u.sr):
```

Comment:
Given L553, the first `if` using `u.Jy` isn't needed?
```suggestion
        if y_unit.is_equivalent(u.Jy / u.sr):
```